test/common/stats/isolated_store_impl_test.cc

```diff
@@ -36,6 +36,15 @@ TEST_F(StatsIsolatedStoreImplTest, All) {
   EXPECT_EQ(0, c1.tags().size());
   EXPECT_EQ(0, c1.tags().size());
 
+  StatNameManagedStorage c1_name("c1", store_.symbolTable());
+  c1.add(100);
+  auto found_counter = store_.findCounter(c1_name.statName());
+  ASSERT_TRUE(found_counter.has_value());
+  EXPECT_EQ(&c1, &found_counter->get());
+  EXPECT_EQ(100, found_counter->get().value());
+  c1.add(100);
+  EXPECT_EQ(200, found_counter->get().value());
```

Comment:
can you also add coverage for find commands returning nullotp for the 3 cases?

Reply:
Added